### PR TITLE
[APP-2533] small miss in UI

### PR DESCRIPTION
**Pull request by Culurio**
**What does this PR do?**

This PR aims to fix some small misses in UI

**Database changed?**

  No

**Where should the reviewer start?**

- [ ] MyGamesBundleView.kt

**How should this be manually tested?**

  Flow on how to test this or QA Tickets related to this use-case: [APP-2533](https://aptoide.atlassian.net/browse/APP-2533)

**What are the relevant tickets?**

  Tickets related to this pull-request: [APP-2533](https://aptoide.atlassian.net/browse/APP-2533)

**Questions:**

   Does this add new dependencies which need to be added? (Eg. new keys, etc.) 

**Code Review Checklist**

- [ ] Documentation on public interfaces
- [ ] Database changed? If yes - Migration?
- [ ] Remove comments & unused code & forgotten testing Logs
- [ ] Codestyle
- [ ] New Kotlin code has unit tests
- [ ] New flows in presenters unit tests
- [ ] Mappers/Validators with any kind of logic unit tests
- [ ] Functional tests pass

[APP-2533]: https://aptoide.atlassian.net/browse/APP-2533?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ
[APP-2533]: https://aptoide.atlassian.net/browse/APP-2533?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Comment by Culurio** [comment]
> Can one of the admins verify this patch?
Clicked enter by mistake when editing the name of the PR

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MyGamesBundleView.kt [hunk 1]
```diff
@@ -92,8 +92,9 @@ fun MyGamesBundleViewContent(
       BundleHeader(
         title = title,
         icon = icon,
-        hasMoreAction = false,
-        titleColor = Palette.White
+        hasMoreAction = true,
+        titleColor = Palette.White,
+          iconColor = Palette.White
```
Comment (ricardo-frango) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MyGamesBundleView.kt:97]: param not aligned

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MyGamesBundleView.kt [hunk 2]
```diff
@@ -92,8 +92,9 @@ fun MyGamesBundleViewContent(
       BundleHeader(
```
Comment (ricardo-frango) [inline app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MyGamesBundleView.kt:92]: "See All" should also be white.

**Review by ricardo-frango — CHANGES_REQUESTED** [review]

File: app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MyGamesBundleView.kt (hunk 2 above)
Comment (Culurio) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MyGamesBundleView.kt:92]: I tested dark and light theme and i belive that the see all text not being in white is a bug in preview

Comment (ricardo-frango) [reply app-games/src/main/java/com/aptoide/android/aptoidegames/feature_apps/presentation/MyGamesBundleView.kt:92]: tested and confirmed

**Pull request merged**